reflex

```diff
@@ -0,0 +1 @@
+-s -r \.go$ -R ^\.go/ -R ^cmd/ -R _test\.go$ make debug
```

Comment:
As reflex is hardly used in ONS repo's ...
Any chance you could add comments into the reflex file detailing what those flags are doing (to save everyones time who looks at this file) - Thanks.

Reply:
Sure